config/enabled.yml

```diff
@@ -1523,6 +1523,10 @@ Lint/Void:
   Description: 'Possible use of operator/literal/variable in void context.'
   Enabled: true
 
+Lint/NumberConvertion:
+  Description: 'Checks usage of `#to_i` for number convertion'
+  Enabled: true
```

Comment:
Should this list is sorted in alphabetical?

Reply:
Yes. 🙂